bgpd/bgp_zebra.c

```diff
@@ -2042,6 +2046,38 @@ static void bgp_zebra_process_local_ip_prefix(int cmd, struct zclient *zclient,
 	}
 }
 
+static void bgp_zebra_process_label_chunk(
+	int cmd,
+	struct zclient *zclient,
+	zebra_size_t length,
+	vrf_id_t vrf_id)
+{
+	struct stream *s = NULL;
+	uint8_t response_keep;
+	uint32_t first;
+	uint32_t last;
+
+	s = zclient->ibuf;
+	response_keep = stream_getc(s);
```

Comment:
I would recommend switching over to using STREAM_GETX functions to prevent a crash from happening and to allow a graceful handling of the stream being in error

Reply:
agree with Donald.